agent/build.gradle

```diff
@@ -78,23 +82,13 @@ startShadowScripts {
 	applicationName = 'convert'
 }
 
-task doc(type: Exec) {
-	def outputDir = file('build/doc')
-	outputs.dir outputDir
-
-	doFirst {
-		mkdir outputDir
-	}
-	commandLine 'pandoc', 'USAGE.md', '-o', "$outputDir/userguide.pdf"
-}
-
 distributions {
 	shadow {
 		baseName = 'teamscale-jacoco-agent'
 		contents {
-			from(doc) {
-				include '**'
+			from(usageToPdf) {
 				into 'documentation'
+				rename 'USAGE.pdf', 'userguide.pdf'
```

Comment:
we should rename USAGE.md to userguide.md and put it in a "documentation" folder. The name USAGE is a renmant of  the distant past :P

Reply:
you can check the fileupload-service repo for how to do that